src/nodes/time.ts

```diff
@@ -53,6 +54,8 @@ const time: MyNodeSpec<Attrs> = {
       },
     },
   ],
+  attrsFromMdastToken: () => ({ datetime: null }),
+  toMyst: (props): Text => ({ type: 'text', value: formatDatetime(props.datetime).f }),
```

Comment:
We can probably introduce a time prop here that is the ISO formatted datetime string.